src/Logic/UiUtil.cs

```diff
@@ -421,6 +421,15 @@ public static void ApplyControlBackspace(TextBox textBox)
                         if (openingBracketIndex >= 0 && text.IndexOf('>', openingBracketIndex + 1) == textIndex)
                             deleteFrom = openingBracketIndex; // delete whole tag
                     }
+                    // Remove entire Ssa / Ass Tags when Ctrl+BackSpace are pressed.
+                    if (index > 0 && text[textIndex] == '}')
+                    {
+                        var startIdx = text.LastIndexOf('{', textIndex - 1);
+                        if (startIdx >= 0 & text.IndexOf('}', startIdx + 1) == textIndex)
+                        {
+                            deleteFrom = startIdx;
+                        }
+                    }

```

Comment:
Perhaps, put them together, because they are almost identical?

```C#
                    if (index > 0)
                    {
                        if (text[textIndex] == '>') // HTML tag?
                        {
                            var openingBracketIndex = text.LastIndexOf('<', textIndex - 1);
                            if (openingBracketIndex >= 0 && text.IndexOf('>', openingBracketIndex + 1) == textIndex)
                                deleteFrom = openingBracketIndex; // delete entire HTML tag
                        }
                        else if (text[textIndex] == '}') // Ssa / Ass tag?
                        {
                            var openingBracketIndex = text.LastIndexOf('{', textIndex - 1);
                            if (openingBracketIndex >= 0 && text.IndexOf('}', openingBracketIndex + 1) == textIndex)
                                deleteFrom = openingBracketIndex; // delete entire Ssa / Ass tag
                        }
                    }
```